src/mpc/structs.jl

```diff
@@ -221,27 +252,30 @@ end
 Base.@kwdef struct MPCElectricStorage < AbstractElectricStorage
     size_kw::Float64
     size_kwh::Float64
-    charge_efficiency::Float64 =  0.96 * 0.975^2
-    discharge_efficiency::Float64 =  0.96 * 0.975^2
+    charge_efficiency::Float64 =  0.96 * 0.975^0.5
+    discharge_efficiency::Float64 =  0.96 * 0.975^0.5
     soc_min_fraction::Float64 = 0.2
     soc_init_fraction::Float64 = 0.5
     can_grid_charge::Bool = true
-    grid_charge_efficiency::Float64 = 0.96 * 0.975^2
+    grid_charge_efficiency::Float64 = 0.96 * 0.975^0.5
```

Comment:
I changed this to be tied to charge_efficiency value like in reg ElectricStorage